### ISSUE TITLE
How to deal with Authz?

### ISSUE THREAD

**Issue by barryib** [ideascube/logs#2]

**Comment by mbaechler** [comment]
What do you need to achieve ? I mean, I put a nginx for auth + reverse proxy and it worked pretty well.

**Comment by barryib** [comment]
@mbaechler Right, nginx for authz work pretty well. But it's a question for Authz (Authentication and Authorization) in a general way.

- Do we need authorization?
- Can we or should we use ES plugins to achieve authz?
- Do we need centralized auth?

This issue has been opened to help us to answer a bunch of questions about authz.
